persistence/persistence-api/src/main/java/org/cbioportal/persistence/ClinicalDataRepository.java

```diff
@@ -51,6 +51,10 @@ List<ClinicalData> fetchSampleClinicalDataClinicalTable(List<String> studyIds, L
                                                             Integer pageSize, Integer pageNumber, String searchTerm,
                                                             String sortBy, String direction);
 
+    @Cacheable(cacheResolver = "generalRepositoryCacheResolver", condition = "@cacheEnabledConfig.getEnabled()")
+    Integer fetchSampleClinicalDataClinicalTableCount(List<String> studyIds, List<String> ids, String searchTerm, 
```

Comment:
Do you think `fetchSampleClinicalTableCount` could be a better name?
Same for `fetchSampleClinicalDataClinicalTable` -> `fetchSampleClinicalTable`

Reply:
Good suggestion, renamed in [9d7c2b3](https://github.com/cBioPortal/cbioportal/commit/9d7c2b38fba7b0573088fd403706ac4ab035d5e7)